core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1113,6 +1113,7 @@ table.tl_listing {
 }
 
 .tl_file, .tl_file_list {
+	position: relative;
```

Comment:
This change seems unrelated. What does it do?

Reply:
it is not unrelated. It adds relative positioning for comments (mode 1) to fix the expand/collapse button. Try removing it in the comments list when this PR is applied (in the Contao demo).